### PR TITLE
Increase severity of various log messages

### PR DESCRIPTION
**Pull request by StephanErb**
By increasing the log severity for log statements outside of the happy path, we can make it significantly simpler to search for problems within a central log aggregator (e.g. elasticsearch)

### CODE REVIEW

File: distributed/scheduler.py [hunk 1]
```diff
@@ -907,7 +907,7 @@ def __init__(
             try:
                 from distributed.dashboard import BokehScheduler
             except ImportError:
-                logger.debug("To start diagnostics web server please install Bokeh")
+                logger.warning("To start diagnostics web server please install Bokeh")
```
Comment (TomAugspurger) [inline distributed/scheduler.py]: Perhaps keep at debug, since bokeh is an optional dependency.

File: distributed/worker.py [hunk 2]
```diff
@@ -585,7 +585,7 @@ def __init__(
             try:
                 from distributed.dashboard import BokehWorker
             except ImportError:
-                logger.debug("To start diagnostics web server please install Bokeh")
+                logger.warning("To start diagnostics web server please install Bokeh")
```
Comment (TomAugspurger) [inline distributed/worker.py]: Prefer to keep at `debug`.

File: distributed/worker.py [hunk 3]
```diff
@@ -1191,7 +1191,7 @@ def send_to_worker(self, address, msg):
             max_connections is not False
             and self.outgoing_current_count >= max_connections
         ):
-            logger.debug(
+            logger.warning(
```
Comment (TomAugspurger) [inline distributed/worker.py]: In https://github.com/dask/distributed/pull/3071#discussion_r330066652 we decided to leave this at debug level. Ideally these warnings would be throttled at a higher-level, but right now they'll show up frequently.

Comment (StephanErb) [reply distributed/worker.py]: Thanks for the quick feedback. I have to admit this was the one why I created this PR in the first place. It sounded too important to be not visible at all.

**Comment by StephanErb** [comment]
Your comment indicates we need another fix for that. Closing fow now to not bother you with a half-done PR. I might come back :)

**Pull request closed without merging**